runtime/doc/vim_diff.txt

```diff
@@ -189,6 +189,7 @@ Input/Mappings:
 
 Normal commands:
   |gO| shows a filetype-defined "outline" of the current buffer.
+  Y now behaves as y$
```

Comment:
we need a new section for `Default mappings:` .

Also weird to say "behaves as", that is vague for no reason. We would instead say "is mapped to", but don't need to since it will be clear by its presence in the `Default mappings:` section.